docs/DeveloperGuide.adoc

```diff
@@ -331,32 +331,146 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: UC01 - Add an order
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User key in the order details.
+2.  Delino add the order details.
+3.  Delino displays order added.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. Delino detects invalid syntax.
+[none]
+** 1a1. Delino shows an error message.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+[discrete]
+=== Use case: UC02 - Clear all orders
+
+*MSS*
+
+1.  User requests to clear all orders.
+2.  Delino clear all existing orders.
+3.  Delino displays order cleared message.
 +
+Use case ends.
+
+*Extensions*
+
 [none]
-** 3a1. AddressBook shows an error message.
+* 1a. Delino detects invalid syntax.
+[none]
+** 1a1. Delino shows an error message.
 +
-Use case resumes at step 2.
+Use case ends.
 
-_{More to be added}_
+[none]
+* 1b. Delino detects no orders.
+[none]
+** 1b1. Delino shows no order to be cleared message.
++
+Use case ends.
+
+[discrete]
+=== Use case: UC03 - Delete an order
+
+*MSS*
+
+1.  User requests to list orders (UC10).
```

Comment:
Does this mean that the user has to key in the command to list the orders before the user can delete a specific order? If that is the case, maybe you guys can remove step one, and leave it up to the user whether he or she wants to see the list before deleting an order.